feat(changelog): elastic-metal-added-ubuntu-2404-lts-noble-numbat-now-av 2024-04-29 (#3131)
scaleway/docs-content: changelog/april2024/2024-04-29-elastic-metal-added-ubuntu-2404-lts-noble-numbat-now-av.mdx

---
title: Ubuntu 24.04 LTS (Noble Numbat) now available!
status: added
author:
    fullname: 'Join the #elastic-metal channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-04-29
category: bare-metal
product: elastic-metal
---

> The freshly released Ubuntu 24.04 LTS is now available when you install your Elastic Metal server ✨

Reviewer: ```suggestion
The latest Ubuntu 24.04 LTS is now available upon installation of your Elastic Metal server ✨
```
